projects/js-packages/publicize-components/src/components/post-publish-share-status/index.tsx

```diff
@@ -1,29 +1,38 @@
 import { useSelect } from '@wordpress/data';
 import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { store as editorStore } from '@wordpress/editor';
 import { store as socialStore } from '../../social-store';
-import { ShareStatusModal } from '../share-status-modal';
+import { ShareStatus } from './share-status';
 
 /**
  * Post publish share status component.
  *
  * @return {import('react').ReactNode} - Post publish share status component.
  */
 export function PostPublishShareStatus() {
-	const { featureFlags } = useSelect( select => {
+	const { featureFlags, postId, isPostPublised, willPostBeShared } = useSelect( select => {
 		const store = select( socialStore );
+
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- `@wordpress/editor` is a nightmare to work with TypeScript
+		const _editorStore = select( editorStore ) as any;
+
+		const meta = _editorStore.getEditedPostAttribute( 'meta' );
```

Comment:
Maybe we could use the [usePostMeta](https://github.com/Automattic/jetpack/blob/efde64628a1d2c57cd796516b94ccc8a1c837ad7/projects/js-packages/publicize-components/src/hooks/use-post-meta/index.js) hook we have to get the `isPublicizeEnabled`

Reply:
Done. Thanks